eo-maven-plugin/src/main/java/org/eolang/maven/optimization/Optimization.java

```diff
@@ -32,5 +34,20 @@
  * @since 0.28.11
  */
 @FunctionalInterface
-public interface Optimization extends Function<XML, XML> {
+public interface Optimization extends ThrowingFunction<Path, XML, FileNotFoundException> {
+
+    /**
+     * Return function if it works without exception.
+     *
+     * @return The function result
+     */
+    default Function<Path, XML> unchecked() {
```

Comment:
@Yanich96 I'm afraid it might be dangerous for the code maintainability. 

From [this](https://www.yegor256.com/2017/10/03/java9.html):
> [Default methods](https://docs.oracle.com/javase/tutorial/java/IandI/defaultmethods.html) were also added to interfaces, which turned types into libraries of code. Types into libraries! It’s even worse than [implementation inheritance](https://www.yegor256.com/2016/09/13/inheritance-is-procedural.html), if you ask me.

Moreover, "default" methods are similar to mixins in other languages (please, correct me if I'm wrong), which also isn't a good feature for maintainability and OOP design. You can read more about it [here](https://www.yegor256.com/2017/03/07/traits-and-mixins.html).

Maybe we can remove this "default" behaviour somehow? What do you think?